src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -37,6 +37,12 @@ export const getBootsrapSrc = queryString => {
   return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap.js`;
 };
 
+export const getBootsrapLegacySrc = queryString => {
+  const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
+  const params = useLiveSrc ? '' : 'test/';
+  return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap-legacy.js`;
```

Comment:
Is is better to have this as an end variable? One for test and one for live. Then only override if the QS is set

Reply:
Also... should this be included in this PR - might be better to keep ads stuff separate from remote toggles for the moment

Reply:
This change and the module/nomodule change could be a small PR that goes in before this PR?

Reply:
It's also here instead of a variable because of the query string override that can be used to test again the live version. Though I'm not sure how much switching we'll be doing, we're planning to go over the test plan before it's allowed on live.

Reply:
Removed this card and raised it as a different issue 👍  https://github.com/bbc/simorgh/issues/7313